mitodl/mitxonline
# fix profile validation and regex error

# What are the relevant tickets?




https://github.com/mitodl/mitxonline/issues/1813
https://github.com/mitodl/mitxonline/issues/1788

# Description (What does it do?)

This PR fixes regex error in console. The regex error is related to breaking changes in https://github.com/tc39/proposal-regexp-v-flag#how-is-the-v-flag-different-from-the-u-flag for special character ( ) [ ] { } /

# How can this be tested?

Make sure no console error for form validation and you get default error message and focus if name field are entered incorrectly
- Test create an account
- Test update your existing profile

# Additional Context



Currently `first_name` and `last_name` use build-in pattern for regex validation, that regex error from console is due to HTML pattern now uses RegExp v flag instead of u flag ([link](https://chromestatus.com/feature/5149507107422208)), some previously valid patterns are now errors e.g. ( ) [ { } requires escaping inside character class
